### PR TITLE
Accessibility 2021 chapter

### PR DESCRIPTION
**Pull request by alextait1**
Progress #2147 

Staging link: https://20211118t121037-dot-webalmanac.uk.r.appspot.com/en/2021/accessibility

WIP! 

Remaining tasks:
- [x] TODOs throughout (mainly charts, tables and internal section linking)
- [x] Add in all external links
- [x] Formatting (back-ticks, straight quotes, check headings and lists etc)
- [x] Check for github style markdown conformance: https://docs.github.com/en/github/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax
- [x] Add some more figures (that middle section is just a big wall of text)
- [x] Add figure descriptions
- [x] Add missing figure gids and SQL (might need @obto 's help with that as tried but failed to find some of them)
- [x] Add figure images (can automatically do this at the end once the figures have died down).
- [ ] Edit it (that might be done after this is merged)

Contributions welcome on this work in progress

### CODE REVIEW

**Comment by alextait1** [comment]
@rviscomi and @tunetheweb this is work in progress, not ready for editing. I'll be spending some more time on it tomorrow night

**Comment by tunetheweb** [comment]
I did some clean up of the markup, added the figures that were in the document, and staged it here:

https://20211118t121037-dot-webalmanac.uk.r.appspot.com/en/2021/accessibility

I think the main things still to do are:
- Add the links that seem to have got lost in the conversion
- Add some more figures (that middle section is just a big wall of text)
- Add figure descriptions
- Add missing figure gids and SQL (might need @obto 's help with that as tried but failed to find some of them)
- Search document for TODO and finish off
- Add figure images (can automatically do this at the end once the figures have died down).
- Edit it (that might be done after this is merged)

Will leave it back over to you for a bit.

**Comment by alextait1** [comment]
> I did some clean up of the markup, added the figures that were in the document, and staged it here:
> 
> https://20211118t121037-dot-webalmanac.uk.r.appspot.com/en/2021/accessibility
> 
> I think the main things still to do are:
> 
> * Add the links that seem to have got lost in the conversion
> * Add some more figures (that middle section is just a big wall of text)
> * Add figure descriptions
> * Add missing figure gids and SQL (might need @obto 's help with that as tried but failed to find some of them)
> * Search document for TODO and finish off
> * Add figure images (can automatically do this at the end once the figures have died down).
> * Edit it (that might be done after this is merged)
> 
> Will leave it back over to you for a bit.

Thanks for this - huge help

File: src/content/en/2021/accessibility.md [hunk 1]
```diff
@@ -17,4 +23,818 @@ featured_stat_3: TODO
...
+  sql_file="../css/focus_outline_0.sql"
+) }}
+
+Often times, default focus indication is removed from interactive content such as buttons, form controls, and links using the CSS property `:focus { outline: none; }` or `:focus { outline: 0; }`, sometimes in conjunction with `:focus-within` and/or `:focus-visible`. We found that 91% of desktop pages have `:focus { outline: 0; }` declared. In some cases, it is removed so that a more effective custom style can be applied. Unfortunately, in many cases it is simply removed and never replaced, which can render a page unusable for keyboard users.
+
+For more information about how to achieve accessible focus indication including some limitations of browser default focus styles, we recommend Sara Soueidan's article, <a hreflang="en" href="https://www.sarasoueidan.com/blog/focus-indicators/">"A guide to designing accessible, WCAG-compliant focus indicators"</a>.
+
+### Prefers-color-scheme and high contrast support
+
+{# TODO authors - can we think of a better name for this section? It covers more that just what's in there now #}
```
Comment (shantsis) [inline src/content/en/2021/accessibility.md:149]: Maybe we call this User preference media queries support? Or Support for user preference media queries?

**Comment by tunetheweb** [comment]
OK let's merge this. I'll do a separate PR for copy editing (mainly spell checking and the like since we've covered the majority of the other technical things I look for in this PR).

**Pull request merged**